gcc/rust/checks/errors/borrowck/rust-bir.h

```diff
@@ -77,26 +77,40 @@ class Statement
   // otherwise: <unused>
   std::unique_ptr<AbstractExpr> expr;
   TyTy::BaseType *type;
+  // stores location of the actual expression from source code
+  // currently only available when kind == Kind::ASSIGNMENT
+  tl::optional<location_t> location;
```

Comment:
do we want to make this optional? if this is just a temporary measure, then we should add a `FIXME` comment

Reply:
If this is a TODO, it would make more sense to use just location and init it to UNKNOWN_LOCATION.

Reply:
ok, will fix that